src/main/groovy/de/dkfz/roddy/execution/jobs/direct/synchronousexecution/DirectCommand.groovy

```diff
@@ -23,21 +23,14 @@ import static de.dkfz.roddy.StringConstants.DOLLAR_LEFTBRACE
 public class DirectCommand extends Command {
```

Comment:
I also readded support for the Direct execution version. Please check, that you do not remove the wrong things!